text/0000-fabric-website.md

```diff
@@ -0,0 +1,57 @@
+
+# Motivation
+[motivation]: #motivation
+
+Presently, there are multiple different places to get started with Hyperledger Fabric. It can be difficult for a new user to know which resource is the best one for them to start with based on what they would like to achieve. The available documentation has excellent content for getting started and explains everything well, but there is also a large amount of additional content on other relevant topics (for example, performance testing) that is also useful to new users.
+
+This website aims to surface this content and bring it with the existing documentation into one easy to consume place for users.
+
+There is an existing website at https://www.hyperledger.org/use/fabric that serves as a landing page for people interested in Fabric. However this has several limitations - namely that it is based on a template that is used for all Hyperledger projects, and is unable to be extended by the fabric maintainers.
+
+Other Hyperledger projects, such as [Sawtooth](https://sawtooth.hyperledger.org/), already have websites similar to what this RFC proposes, that collate all relevant resources into one place.  Adding a website with a similar purpose for Fabric will serve as a "front door" to the project, and give it a more approachable and inviting feel.
+
+This website is for any user of Hyperledger Fabric, whether they are a developer of smart contracts or client applications, a network operator, or someone wishing to get involved in the Fabric community. It is for both new users wishing to get started with Fabric and existing users wanting to have a single place to access the resources they use most.
+
+The expected outcome for this project is to have a fully functioning website for our users, which is continually being updated with the latest and best resources.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The current proposed structure for the website is that it will be made up of four primary sections, separated by tabs. These sections are based on the different types of Fabric user that could use this website as a resource. The sections are:
+
+- An **Overview** tab, that will include general information about Hyperledger Fabric and what it is;
+- An **Operators** tab, that will include an explanation of the role of a network operator and links to resources this type of user might find most useful, such as the operations guide and deployment guide;
+- A **Developers** tab, that will include an explanation of the role of a developer and links to resources this type of user might find most useful, such as the VS Code extension and developer tooling, the commercial paper sample, and the latest chaincode and gateway SDK versions;
+- A **Community** tab, that will include links to the most relevant resources for contributing to Fabric, such as the main Fabric gitHub repository, the Jira dashboard, and the contributing guide.
```

Comment:
Rather than linking to the README and pointing people to the other links in the Community section of the README, maybe the user experience here would feel better to just have additional links directly from this site to the more relevant community links.